.editorconfig

```diff
@@ -1,6 +1,69 @@
 # top-most EditorConfig file
 root = true
 
-[*.cs]
+[*.{cs,vb}]
 indent_style = space
-indent_size = 4
\ No newline at end of file
+indent_size = 4
+
+# Organize usings
+dotnet_sort_system_directives_first = true 
```

Comment:
When sorting using directives, everything inside `System` should come first. Is that correct?

Reply:
There is no option for this, they always come last 😉